websocket/handler_test.go

```diff
@@ -44,20 +45,20 @@ func TestPingHandler(t *testing.T) {
 
 func TestHandler_HandlerWSEvents(t *testing.T) {
 	// ---- Setup ----
-	hlr := &Handler{
-		websocketUpgrader: websocket.Upgrader{
-			ReadBufferSize:  10240,
-			WriteBufferSize: 10240,
-			CheckOrigin: func(r *http.Request) bool {
-				return true
-			},
-		},
-		user:              NewUserStore(2),
-		bufferChannel:     make(chan EventsBatch, 10),
+	upgrader := connection.NewUpgrader(connection.UpgraderConfig{
+		ReadBufferSize:    10240,
+		WriteBufferSize:   10240,
+		CheckOrigin:       false,
+		MaxUser:           2,
 		PongWaitInterval:  time.Duration(60 * time.Second),
 		WriteWaitInterval: time.Duration(5 * time.Second),
-		PingChannel:       make(chan connection, 100),
-		UniqConnIDHeader:  "x-user-id",
+		ConnIDHeader:      "x-user-id",
+		ConnGroupHeader:   "string",
+	})
+	hlr := &Handler{
+		upgrader:      upgrader,
+		bufferChannel: make(chan EventsBatch, 10),
```

Comment:
We can make these values configurable too.

Reply:
It is configurable. This is just for test

Reply:
ohh ok